services/gateway/shared/customMapper.js

```diff
@@ -70,6 +73,7 @@ const mapObject = (rootObj, definition, subObj = rootObj) => Object.keys(definit
 			const tempObj = mapObject(rootObj, definition[key], subObj[key]);
 			const values = Object.values(tempObj);
 			if (objWithValues(values) && validate(tempObj)) acc[key] = tempObj;
```

Comment:
We can remove `validate` function since `objWithValues` function already takes care of it.
```suggestion
			if (objWithValues(values)) acc[key] = tempObj;
```

Reply:
Nice catch 💯 Applied both.